tests/Core/Config/GeneratorArgTest.php

```diff
@@ -23,37 +23,59 @@ final class GeneratorArgTest extends TestCase
     /**
      * Ensure that the generator property is set when the parameter is passed a valid value.
      *
-     * @param string $generatorName Generator name.
+     * @param string $argumentValue         Generator name passed in the command line.
+     * @param string $expectedPropertyValue Expected value of the generator property.
      *
-     * @dataProvider dataGeneratorNames
+     * @dataProvider dataValidGeneratorNames
      *
      * @return void
      */
-    public function testGenerators($generatorName)
+    public function testValidGenerators($argumentValue, $expectedPropertyValue)
     {
-        $config = new ConfigDouble(["--generator=$generatorName"]);
+        $config = new ConfigDouble(["--generator=$argumentValue"]);
 
-        $this->assertSame($generatorName, $config->generator);
+        $this->assertSame($expectedPropertyValue, $config->generator);
 
-    }//end testGenerators()
+    }//end testValidGenerators()
 
 
     /**
-     * Data provider for testGenerators().
+     * Data provider for testValidGenerators().
      *
-     * @see self::testGenerators()
+     * @see self::testValidGenerators()
      *
      * @return array<int, array<string>>
      */
-    public static function dataGeneratorNames()
+    public static function dataValidGeneratorNames()
     {
         return [
```

Comment:
As this array is now a little more involved, I think it would make it more straight-forward to maintain and to read the tests if the array included test case names + argument names.

Reply:
Done